requirements.txt

```diff
@@ -1,5 +1,5 @@
 pvlib>=0.9.0,<0.10.0
-shapely>=1.6.4.post2
+shapely>=1.6.4.post2,<=1.7.1
```

Comment:
Could you change this to `shapely>=1.6.4.post2,<2`?

shapely 1.8.0 is now out, and I don't see any reason to restrict consumers to versions that shouldn't break.

Reply:
Ok, done 👍